bindings/python/iota_sdk/types/unlock_condition.py

```diff
@@ -104,7 +114,10 @@ class StateControllerAddressUnlockCondition(UnlockCondition):
     Args:
         address: The state controller address that owns the output.
     """
-    address: Union[Ed25519Address, AccountAddress, NFTAddress]
+    address: Union[Ed25519Address, AccountAddress, NFTAddress] = field(
```

Comment:
Just wondering, is there no way to typedef the union itself so we don't have to repeat it all the time?

Reply:
One could create an alias like this
```
from typing import TypeAlias
AddressUnion: TypeAlias = Union[Ed25519Address, AccountAddress, NFTAddress]
```

Reply:
If you think it makes it better, yeah. Easier to change in only one place I guess. Less verbose.

Reply:
Agreed

Reply:
Done

Reply:
But requires python 3.10